doc/release-notes.md

```diff
@@ -123,6 +123,8 @@ New RPCs
   a future release. Note that in either case, the `status` field
   now reflects the status of the current block rather than the next
   block. (#23508)
+- A new `getmempooltxspendingprevout` RPC has been added, which scans the mempool to find
+  transactions spending any of the given outpoints. (#24408)
```

Comment:
Maybe move this to a separate file for now?

`doc/release-notes-24408.md`

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/24408/commits/4c96203ec36f5634d6f73f3b9a1d99bbd91dcc21, let me know if the format should be different